lib/screen.dart

```diff
@@ -9,4 +9,5 @@ class Screen {
   static Future setBrightness(double brightness) =>_channel.invokeMethod('setBrightness',{"brightness" : brightness});
   static Future<bool> get isKeptOn async => (await _channel.invokeMethod('isKeptOn')) as bool;
   static Future keepOn(bool on) => _channel.invokeMethod('keepOn', {"on" : on});
+  static Future resetBrightness() => _channel.invokeMethod('resetBrightness');
```

Comment:
I wonder if a different name would be appropriate. The brightness isn't really "reset", it's more like, the brightness override is removed?

Reply:
disableBrightnessOverride?